docs/usage.rst

```diff
@@ -258,8 +263,14 @@ any unapplied migrations. It is executed in a separate thread, so the main threa
 
 Settings
 ^^^^^^^^
+
 :code:`HURRICANE_VERSION` - is sent together with webhooks to distinguish between different versions.
 
+From version 1.6.0 onwards, all Django Hurricane settings can be set as either environment variables or in the Django settings file.
+The Django settings take precedence over the environment variables. Both of those methods need the settings to be prefixed with `HURRICANE_`.  
```

Comment:
@liquidiert Can you maybe add a hint, that hyphens need to be replaced by underscores and the setting needs to be in all caps? It should be somewhat obvious, but let's just consider the DAU here ;)

Reply:
@tschale I added another paragraph stating "the obvious" :)